### PR TITLE
[improvement](storage-policy) Add check validity when create storage policy.

### PR DESCRIPTION
**Pull request by deardeng**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.16 seconds
 load time: 449 seconds
 storage size: 17172425439 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221118150114_clickbench_pr_48861.html

File: fe/fe-core/src/main/java/org/apache/doris/backup/S3Storage.java [hunk 1]
```diff
@@ -245,6 +245,9 @@ public Status directUpload(String content, String remoteFile) {
         } catch (UserException ue) {
             LOG.error("connect to s3 failed: ", ue);
             return new Status(Status.ErrCode.COMMON_ERROR, "connect to s3 failed: " + ue.getMessage());
+        } catch (Exception e) {
+            LOG.error("connect to s3 failed: ", e);
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/backup/S3Storage.java:249]: Use `warn`, need to modify above all LOG.error to LOG.warn

File: fe/fe-core/src/main/java/org/apache/doris/catalog/S3Resource.java [hunk 2]
```diff
@@ -121,12 +126,51 @@ protected void setProperties(Map<String, String> properties) throws DdlException
...
+
+        String testFile = bucket + properties.getOrDefault(S3_ROOT_PATH, "") + "/test-object-valid.txt";
+        String content = "doris will be better";
+        Status status = storage.directUpload(content, testFile);
+        if (status != Status.OK) {
+            LOG.warn("ping update file status: {}, properties: {}", status, propertiesPing);
+            return false;
+        }
+
+        Status delete = storage.delete(testFile);
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/catalog/S3Resource.java:165]: wrap with `try .. finally`? To make sure the file will be deleted finally.

Comment (deardeng) [reply fe/fe-core/src/main/java/org/apache/doris/catalog/S3Resource.java:165]: fixed

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**